src/app/components/page/landing-page/landing-page.component.html

```diff
@@ -86,10 +86,11 @@ <h2>Documentation</h2>
   </section>
 
   <footer>
+<!--
     <div class="links">
       <div class="link">Contact</div>
     </div>
-
-    <p>PhyloDiver is developed by the <a href="http://vi-bio.ncsa.illinois.edu/" target="_blank">Visual Analytics Group</a> and <a href="https://www.el-kebir.net/" target="_blank">El-Kebir Lab</a> at the <a href="https://illinois.edu/" target="_blank">University of Illinois at Urbana-Champaign</a>, is supported by the <a href="https://cancer.illinois.edu/" target="_blank">Cancer Center of Illinois</a>, and is an open source project available on <span class="nowrap">GitHub: <a href="https://github.com/ncsa/phylodiver-viz" target="_blank">PhyloDiver</a>.</span></p>
+-->
+    <p>PhyloDiver is developed by the <a href="http://vi-bio.ncsa.illinois.edu/" target="_blank">Visual Analytics Group</a> and <a href="https://www.el-kebir.net/" target="_blank">El-Kebir Lab</a> at the <a href="https://illinois.edu/" target="_blank">University of Illinois at Urbana-Champaign</a>, is supported by the <a href="https://cancer.illinois.edu/" target="_blank">Cancer Center of Illinois</a>, and is an open source project available on <a href="https://github.com/ncsa/phylodiver-viz" target="_blank">GitHub</a>. <span class="nowrap">Contact <a href="mailto:visualanalytics@illinois.edu">visualanalytics@illinois.edu</a></span></p>
```

Comment:
@cholson branding question, here and below: better to say "developed by Visual Analytics and the El-Kebir Lab"?

Reply:
hmm, not sure i feel strongly either way…leave it for now?

Reply:
10-4